script.js

```diff
@@ -0,0 +1,55 @@
+//Step One: Fetch Data
+fetch("https://ghibliapi.herokuapp.com/films")
+.then((res) => {
+return res.json();
+})
+.then((films) => {
+  console.log(films);
+
+//Step Two: Use Data to Add Movies to Dropdown
+let movieSelection = document.querySelector("#movie-selection");
+//add each movie to the dropdown
+for (let film of films) {
+    const option = document.createElement("option");
+    option.setAttribute("value", film.title);
+    option.textContent = film.title;
+    movieSelection.append(option);
+  }
+//Step Three: Display Selected Movie Info
+let movieTitle = document.querySelector("#movie-title");
+let releaseDate = document.querySelector("#movie-release-year");
+let movieDescription = document.querySelector("#movie-description");
+
+let selectedMovie;
+//add event listener for when dropdown changes
+//selected movie should be displayed on the screen 
+
+movieSelection.addEventListener("change", () =>{
+    for (let film of films){
+        if(movieSelection.value === film.title){
+            console.log(film);
+            selectedMovie = film;
+            // selectedMovie.textContent = film.name;
+            movieTitle.textContent = film.title;
+            releaseDate.textContent = film.release_date;
+            movieDescription.textContent = film.description;
+        }
+    }
+})
+//Step Four: Submit Review
+let submittedReview = document.querySelector("#submitted-review");
+let form = document.querySelector("form");
+//set an event listener on the "Submit Review" button. 
+//Every time this button is clicked, a new list item is created
+//li are added to the ul
+form.addEventListener("submit", (e) => {
+    e.preventDefault();
+    let newReview = document.createElement("li");
+    let review = document.querySelector("#textInput").value;
+    newReview.innerHTML = `<strong>${movieSelection.value}</strong>:</b> ${review}`;
```

Comment:
This line is the reason your last two tests are failing. It's such a small thing and it's really difficult to discern from the error messages in the tests so I'm just going to give you the points for it.
```suggestion
    newReview.innerHTML = `<strong>${movieSelection.value}:</strong></b> ${review}`;
```
the colon just needed to be **inside** the `strong` tag, that's all.